### ISSUE TITLE
Block Request to private ip subnets

### ISSUE THREAD

**Issue by GQDeltex** [GQDeltex/go-http-proxy#1]
Block Requests to e.g.
```
10.0.0.0 - 10.255.255.255
172.16.0.0 - 172.31.255.255
192.168.0.0 - 192.168.255.255
```

**Comment by AnTheMaker** [comment]
That would be perfect!!
On that note: would it be possible to ONLY allow requests to external urls starting with `http://` or `https://` (no other protocols)?

**Comment by GQDeltex** [comment]
closed by cf8a582b9285aacc21294e3c10623b93ee299ee8
